src/detectors/MPGD/MPGD.cc

```diff
@@ -23,30 +23,72 @@ void InitPlugin(JApplication *app) {
     // Digitization
     SiliconTrackerDigiConfig barrel_digi_default_cfg;
     barrel_digi_default_cfg.threshold = 0.25 * dd4hep::keV;
-    barrel_digi_default_cfg.timeResolution = 8;
-    app->Add(new JChainFactoryGeneratorT<SiliconTrackerDigi_factory>({"MPGDBarrelHits"}, "MPGDBarrelDigiHits", barrel_digi_default_cfg));
+    barrel_digi_default_cfg.timeResolution = 10;
+    app->Add(new JChainFactoryGeneratorT<SiliconTrackerDigi_factory>({"MPGDBarrelHits"}, "MPGDBarrelRawHits", barrel_digi_default_cfg));
 
     // Convert raw digitized hits into hits with geometry info (ready for tracking)
     TrackerHitReconstructionConfig tracker_hit_reco_cfg;
-    tracker_hit_reco_cfg.time_resolution = 8;
+    tracker_hit_reco_cfg.time_resolution = 10;
     app->Add(new JChainFactoryGeneratorT<TrackerHitReconstruction_factory>(
-            {"MPGDBarrelDigiHits"},     // Input data collection tags
+            {"MPGDBarrelRawHits"},     // Input data collection tags
             "MPGDBarrelRecHits",    // Output data tag
              tracker_hit_reco_cfg));           // Hit reco default config for factories
 
     // Digitization
     SiliconTrackerDigiConfig dirc_digi_default_cfg;
-    barrel_digi_default_cfg.threshold = 0.25 * dd4hep::keV;
-    barrel_digi_default_cfg.timeResolution = 8;
-    app->Add(new JChainFactoryGeneratorT<SiliconTrackerDigi_factory>({"MPGDDIRCHits"}, "MPGDDIRCDigiHits", dirc_digi_default_cfg));
+    dirc_digi_default_cfg.threshold = 0.25 * dd4hep::keV;
+    dirc_digi_default_cfg.timeResolution = 10;
```

Comment:
This stupid typo mistake could/should have been avoided with a structure like this:
```
    SiliconTrackerDigiConfig dirc_digi_default_cfg {
      .threshold = 0.25 * dd4hep::keV,
      .timeResolution = 10,
    };
```

Reply:
(not your typo, but the pre-existing typo you fixed)

Reply:
Let's postpone this to a different PR.

Reply:
I do like that structure, it is much easier to read!

Reply:
See https://github.com/eic/EICrecon/issues/746

Also, I think the pythonic trailing comma on the `timeResolution` line may not work (or maybe it does... I think this is the one place where C(++) may actually allow trailing commas.)